programming_examples/matrix_scalar_add/README.md

```diff
@@ -0,0 +1,78 @@
+# Matrix Scalar Addition
```

Comment:
I discuss tiling at different layers of abstraction but I don't discuss temporal vs spacial at all here. @jgmelber can you help me understand what spacial/temporal settings for tiling I could add as examples or write about here in the readme?